src/Component/Count.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Count = (props) => {
```

Comment:
You can use destructuring:
```suggestion
const Count = ({ count, onClick }) => {
```